var/spack/repos/builtin/packages/blast-plus/package.py

```diff
@@ -79,6 +80,7 @@ def patch(self):
     depends_on("bzip2", when="+bzip2")
     depends_on("lzo", when="+lzo")
     depends_on("pcre", when="+pcre")
+    depends_on("llvm-openmp", when="%apple-clang")
```

Comment:
Should this include the new `openmp` variant?

```suggestion
    depends_on("llvm-openmp", when="+openmp%apple-clang")
```

Reply:
yeah, just an oversight on my part. the package configure file adds openmp by default, but it wouldn't need this dependency if you ask to build without openmp obviously.